tests/unit_tests/test_unit.py

```
@@ -2489,8 +2489,2505 @@ def test_db_spec2pep_forward_no_cache(tiny_config):
+
+
+def test_version():
+    """Check that the version is not None."""
+    assert casanovo.__version__ is not None
+
+
+@pytest.mark.skip(reason="Skipping due to Linux deadlock issue")
+def test_n_workers(monkeypatch):
+    """Check that n_workers is correct without a GPU."""
+    monkeypatch.setattr("torch.cuda.is_available", lambda: False)
+    cpu_fun = lambda x: ["foo"] * 31
```

Ruff (0.15.10)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `cpu_fun` as a `def`

(E731)